src/swish/mat.ss

```diff
@@ -241,7 +290,8 @@
           (flush-output-port mo-op))
         (for-each
          (lambda (mat/name)
-           (run-mat mat/name reporter incl-tags excl-tags))
+           (parameterize ([mat-annotations '()])
```

Comment:
Should the parameterize of `mat-annotations` be done inside `run-mat` so that `run-mat` can be called from the outside world on mats that use annotations?

Reply:
I pushed a commit that moves the `parameterize`. Since we don't yet export `get-result-meta-data`, the `reporter` procedure passed to `run-mat` doesn't have a good way to harvest the annotations or the other result meta-data at the moment.

I didn't want to make a breaking change to `reporter`. An earlier draft used `procedure/arity?` to check for legacy `reporter` interface, but that seemed awkward.

Reply:
That's OK for now. By moving the parameterize, run-mat will work for mats with annotations, even if there's no good way to get the annotations out.

Reply:
Good thought.